tests/system/test_cluster/test_agent_files_deletion/test_agent_files_deletion.py

```diff
@@ -32,6 +32,7 @@ def register_agent():
     """Restart the removed agent to trigger auto-enrollment."""
     yield
     host_manager.get_host(agent_host).ansible('command', f'service wazuh-agent restart', check=False)
+    sleep(15)
```

Comment:
In this case, maybe it's better to add a fixture to monitor the agent enrollment event in the `ossec.log` file.